### PR TITLE
Automation test - Members/activities - Admins/Members

### PR DESCRIPTION
**Pull request by Micaela-Rossi**
A complete automation test for activities, but still working in the members test.
06-30-2023: Added a few steps for members test, but still working in a function to compare the values in the inputs before the update and after the update.
07-06-2023: Added base for implement log in in members flow, discard the commit to solve problems to merge.
Added the implementation of log in to the members e2e
07-12-2023: Completed the automation test for admin flow using the shared components instead of having it repeated in each page object.

### CODE REVIEW

**Comment by Micaela-Rossi** [comment]
[[QA] Automation Test - Members from Admin flow and Activities from Member flow](https://trello.com/c/mGgbtskB/168-qa-automation-test-members-from-admin-flow-and-activities-from-member-flow)

File: test/pageObjects/logIn.js
Comment (IgnacioCanton) [inline test/pageObjects/logIn.js:1]: I think it would be better if this file goes inside sharedComponents folder

File: wdio.conf.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const allure = require('allure-commandline')
+//const allure = require('allure-commandline')
```
Comment (IgnacioCanton) [inline wdio.conf.js:1]: Some bugs could be created if you comment this line. Why did you comment it?

**Review by IgnacioCanton — CHANGES_REQUESTED** [review]

File: wdio.conf.js (hunk 1 above)
Comment (Micaela-Rossi) [reply wdio.conf.js:1]: really, i forgot, it was a moment solution for something but i can change it back

Comment (Micaela-Rossi) [reply wdio.conf.js:1]: Captura de pantalla 2023-07-13 040013

Comment (Micaela-Rossi) [reply wdio.conf.js:1]: I'm having troubles with the allure, the rest it's changed, tomorrow i will ask for help so can solve and merge

**Comment by Micaela-Rossi** [comment]
Some errors with the files made me close this pr and make a new #92 to make sure not merge errors to master.

**Pull request closed without merging**